cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -95,6 +100,18 @@ function ManagementCredentials(props: Partial<StepProps>) {
             if (goToStep && currentStep && submitForm) {
                 goToStep(currentStep + 1);
                 submitForm(currentStep);
+                awsDispatch({
```

Comment:
I'm curious about the threading of events here. It appears that we're doing `goToStep` first, and then dispatching the message to add the resources and set the selected image. Should we be calling `goToStep` last?

Reply:
Yes. I should notice this one. Will call `goToStep` in the last